src/styles/blocks/header.scss

```diff
@@ -0,0 +1,37 @@
+.header {
+  box-sizing: border-box;
+  background-image: url(../images/bo-mobile.png);
+  background-position: center;
+  background-size: cover;
```

Comment:
Add 'background-repeat: no-repeat;' to prevent the background image from repeating.